docs/codestyle.md

```diff
@@ -0,0 +1,58 @@
+# WikiTree code style
+
+## Code style
+
+we aim for consistency, generality, readability and reduction of git diffs. Similar language constructs are formatted with similar rules. Style configuration options are deliberately limited and rarely added. Previous formatting is taken into account as little as possible.
+
+## Basic rules
+
+| Rule | Value | Note |
+| ---  | :---: | --- |
+| Line length | 120 characters |  |
```

Comment:
Standard in editors is `80` charcters, but this is the relict of old times and terminal editor, where screens were small and fonts were ugly 🙃 - today's screens are wide enough to comfortbaly show `120` characters even with left & right side panel opened. We are using this line lenght at work and it's perfectly OK.

Reply:
I agree; 80 is too short.